numpy/lib/array_utils.py

```diff
@@ -1,3 +1,4 @@
+# flake8: noqa: F401
```

Comment:
not `ruff`?

Reply:
Both work, but of course I need to change to `ruff` for consistency.